### ISSUE TITLE
Bug: mtx-grid. Data in table is not updated automatically

### ISSUE THREAD

**Issue by magoarcano** [ng-matero/extensions#19]
if I have the following column definitions:
```
columns = [
    { header: 'name', field: 'name'},
    { header: 'calories', field: 'calories', formatter: (data: any) => data.calories},
    { header: 'fat', field: 'fat', cellTemplate: this.fatTpl },
    { header: 'carbs', field: 'carbs'},
    { header: 'protein', field: 'protein'},
  ]
```
'name', 'carbs' and 'protein' are not updated when their values changes. Only works with a formatter ('calories') or a template ('fat')

I created a stackblitz (edited an example from angular material library)
https://stackblitz.com/angular/jrkyyjvxqql?file=src%2Fapp%2Fsort-overview-example.ts

There you can see that pressing the 'add' button updated calories values in the original table but not in mtx-grid

**Comment by nzbin** [comment]
I haven't seen the add button.

**Comment by nzbin** [comment]
This issue has been fixed in v10.5.1.
